.travis.yml

```diff
@@ -174,28 +177,30 @@ jobs:
     if: type IN (pull_request)
     env:
     - NODE_ENV=production
-    - BUNDLE_ANALYSIS=1
     - PORT=12345
-    - VERSION=$TRAVIS_PULL_REQUEST_BRANCH
-    - SENTRY_ENVIRONMENT=pr
+    - VERSION=$TRAVIS_BRANCH
+    - SENTRY_ENVIRONMENT=staging
+    - BUNDLE_ANALYSIS=1
     - STREAMR_API_URL=https://staging.streamr.com/api/v1
     - STREAMR_WS_URL=wss://staging.streamr.com/api/v1/ws
     - PLATFORM_ORIGIN_URL=https://staging.streamr.com
     - STREAMR_URL=https://staging.streamr.com
+    - PLATFORM_PUBLIC_PATH=https://marketplace-staging.streamr.com
     - GOOGLE_ANALYTICS_ID=55331565-5
     - STORYBOOK_BASE_PATH=storybook
     - MARKETPLACE_CONTRACT_ADDRESS=0x98af5996A5cbfAC95aE1E860cBD60d472f214Be0
-    - MARKETPLACE_CONTRACT_ADDRESS_OLD=0xA10151D088f6f2705a05d6c83719e99E079A61C1
+    - MARKETPLACE_CONTRACT_ADDRESS_OLD=0x62d08a44d6db7821c9556032a437beb4bd050abd
     - NEW_MP_CONTRACT=on
-    - TOKEN_CONTRACT_ADDRESS=0x0Cf0Ee63788A0849fE5297F3407f701E122cC023
-    - WEB3_REQUIRED_NETWORK_ID=1
-    - WEB3_PUBLIC_HTTP_PROVIDER=https://mainnet.infura.io/v3/17c3985baecb4c4d94a1edc2c4d23206
-    - WEB3_PUBLIC_WS_PROVIDER=wss://mainnet.infura.io/ws/v3/17c3985baecb4c4d94a1edc2c4d23206
+    - DATA_TOKEN_CONTRACT_ADDRESS=0x67a9fda28c4df570d1d5350acfa345194f4d4100
+    - DAI_TOKEN_CONTRACT_ADDRESS=0x4CEed45382203e84d2eAB4A9F37D4F141e79A574
+    - UNISWAP_ADAPTOR_CONTRACT_ADDRESS=0x207c73d29F12e28c4e08C82A9d0f6Fb38CEa60C3
+    - WEB3_REQUIRED_NETWORK_ID=8995
+    - WEB3_PUBLIC_HTTP_PROVIDER=https://staging.streamr.com:8540
+    - WEB3_PUBLIC_WS_PROVIDER=wss://staging.streamr.com:8540/ws
     - SENTRY_DSN=https://e012761929d24baa96ca82772f60b395@sentry.io/1203154
     - LOGROCKET_SLUG=et0uhu/streamr-frontend-local
     - COMMUNITY_PRODUCTS=on
     - STREAMR_ENGINE_NODE_ADDRESSES=0xFCAd0B19bB29D4674531d6f115237E16AfCE377c
-    - COMMUNITY_PRODUCTS=on
```

Comment:
no community products on PR Branches?

Reply:
It's there on line 202, this got thrown by the copy-paste